README.md

```diff
@@ -9,6 +9,8 @@ The notebooks can be opened through google colab or jupyter lab. To do this thro
 
 You can save a copy of the file in your personal google drive to make edits and further examine the data. The notebooks can also be used as the basis for understanding how to examine and analyze ISONE data. 
 
+The files can be updated by creating a pull request for your edits and having it reviewed by a member of the UCS organization to be incorporated into the main branch of the repository. 
```

Comment:
The best way to explain this is by having a `CONTRIBUTING.md` file. Here's an [example](https://github.com/arfc/osier/blob/main/docs/source/contrib.md) from one of my projects. 

I will create an issue and you can update this pull request (or the next one) to have a file describing the change. The `README.md` can point to the contributing guide.